### ISSUE TITLE
Revise existing Gantt Chart

### ISSUE THREAD

**Issue by onikh** [CSAllenISD/2023-ISP-ShelterSafe#47]

**Comment by onikh** [comment]
This is something we should get on ASAP, and it gives us a way to assign tasks for each person efficiently with color coding.

**Comment by omerert** [comment]
We will need this to meet up with recent updates

**Comment by kripatailor** [comment]
I agree, it we update the Gantt chart as frequently as we can it would be easy for us to know what all we have left to do.
